tests/test_categorized_view.py

```diff
@@ -0,0 +1,33 @@
+import os
+from unittest import TestCase
+import xml.etree.ElementTree as ET
+from builder.converter.converter import convert_to_xml
+
+
+class TestCategorizedView(TestCase):
```

Comment:
lack of tests to confirm proper transformation of example YAML to expected fixture output

please extend